readme.md

```diff
@@ -109,18 +113,27 @@ Let's go through the above code examples line-by-line...
 
 #### Selecting the Element
 
-We want our "click handler" -- what we're calling an event listener -- to trigger every time the button is clicked. In order for this to happen, we need to represent that button in Javascript. We can "select" that button using a Javascript selector like [`.querySelector()`](http://www.w3schools.com/jsref/met_document_queryselector.asp). You'll learn more about these in the DOM class.
+We want our "click handler" -- what we're calling an event listener -- to trigger every time the button is clicked. In order for this to happen, we need to select the button from the DOM and represent that button in Javascript.
+
+<details>
+<summary>
+Selecting DOM elements with Vanilla JS & jQuery
+</summary>
+In vanilla javascript, we would use `.querySelector()`, however in jQuery we ***wrap it in ca$h*** using the `$()` function which has some added functionality than its vanilla js counterpart. If `.querySelector()` were a kitchen knife, `$()` would be a swiss army knife or a multi-tool.
+</details>
```

Comment:
You're probably going to mention it in lesson, but i would put some framing in here about how there is the JQ way and the vanilla JS way to do things. And how this markdown file is setup to include both